### ISSUE TITLE
[Data viz] Color palettes page enhancement

### ISSUE THREAD

**Issue by jeanservaas** [carbon-design-system/design-language-website#564]
There are several unique components in this page that will either require a modification of the Gatsby Tabs component or a net new component. I've highlighted and spec'd each of the new components and also shown the page in context.

### New Color Palettes page in context

New components are highlighted

![[data_viz]color_palette_context.png](https://images.zenhubusercontent.com/5b96e742f28d730b01a470c0/e1f21a9b-148c-403a-ab02-773641fd37ff)

### 1. Categorial palette component and 4. Alert palette component are the same

This is basically just the Gatsby tab component with a few exceptions:

- 48px tall color swatches have 1px stroke (top, bottom and sides); for the light theme, the stroke will be #ffffff for the dark theme the stroke color will be #171717

- Rather than having the content dictate the width of the tabs, Mike wants the width of both tabs to span four columns and match the width of the first column of swatches. 

![image.png](https://images.zenhubusercontent.com/5b96e742f28d730b01a470c0/346971df-2d9b-4cb8-aa75-b4febdbf96e2)

**Categorical palette  and alert palette component for light and dark themes at 1312**

![Categorical_palette_component.png](https://images.zenhubusercontent.com/5b96e742f28d730b01a470c0/0921d392-6dfe-4d0d-90a3-6a2625f67429)

### 2. Color combinations commponent specs

This is similar to the Gatsby tab component, but combines the tabs with a dropdown... a combination that we haven't done yet in the Gatsby theme. @vince thinks it will be easier to build it from scratch than hack the Gatsby tab components.

- The dropdown will have five options (1-color group, 2-color-group, 3-color-group, 4-color group, 5-color group) and will default to "1-color group"

- These group values will all vary depending on whether you're designing for a dark or a light theme

- As with the above component, the tabs will span four columns and the dropdown will span four columns

**Color combo component specs at 1312**

![Color_combinations_component.png](https://images.zenhubusercontent.com/5b96e742f28d730b01a470c0/e8248c70-2fab-42d0-9710-8d7cfae8fb24)

### 3. Sequential and diverging palette component specs

For BOTH the sequential and diverging palettes there is NO difference between light and dark themes. So there is no need to have tabs that toggle between light and dark. So there will be no white or dark stroking around the steps... only ONE 1px #ffffff stroke down the middle

This component should function exactly like the color palette component on the IDL site https://www.ibm.com/design/language/color#specifications — with two small exceptions 

- There are only two columns, not three

- The two tabs will span 4 columns

- The Continuous tab shows a gradient... not stepped values; Eliad may have some insight on how to do this programatically

![image.png](https://images.zenhubusercontent.com/5b96e742f28d730b01a470c0/964ba7a9-54d8-45b9-b12d-2c822763f1a1)

![Sequential&diverging_component.png](https://images.zenhubusercontent.com/5b96e742f28d730b01a470c0/9b5d9bcb-1ee4-4c59-8f71-abb5b2dfcf97)

## Breakpoints

1056
![image.png](https://images.zenhubusercontent.com/5b96e742f28d730b01a470c0/96bd7e2b-7ca7-4f61-8e5b-3e83261d3793)

<br>

672

![image.png](https://images.zenhubusercontent.com/5b96e742f28d730b01a470c0/210596ee-6b34-41fc-808f-6210bac719f1)

<br>

Mobile

![image.png](https://images.zenhubusercontent.com/5b96e742f28d730b01a470c0/ba982ebb-27d9-4d09-b9ad-88e189058db1)

![image.png](https://images.zenhubusercontent.com/5b96e742f28d730b01a470c0/b3830540-1a71-4b94-9fcb-774a56cb9932)

**Comment by jeanservaas** [comment]
@jnm2377 

![Sequential_palette_Component_revised.png](https://images.zenhubusercontent.com/5b96e742f28d730b01a470c0/bd2e56c8-1acc-4414-b9cb-35948a671c9b)

After talking with the Cognos team about some palette inconsistencies, Sadek and I added two more monochrome palettes to the sequential palettes page.

We also noticed that it was inconsistent not to label these palettes as "options" like the color pairings component does... so that is added here as well.

I hope this isn't a big curve ball... sorry! We can talk through this too sometime this week.

ps, i have NO IDEA why all these images are broken... will look into that shizz tomorrow.
